frontend/src/lib/components/LemonCalendar/LemonCalendar.tsx

```diff
@@ -0,0 +1,141 @@
+            setFirstMonth(props.firstMonth)
+        }
+    }, [props.firstMonth])
+
+    return (
+        <div className="LemonCalendar flex items-start gap-4">
+            {range(0, months).map((month) => {
+                const startOfMonth = (firstMonth ? dayjs(firstMonth) : dayjs()).add(month, 'month').startOf('month')
+                const endOfMonth = (firstMonth ? dayjs(firstMonth) : dayjs()).add(month, 'month').endOf('month')
+                const stringMonth = startOfMonth.format('YYYY-MM-DD')
+                // TODO: support the easier US Sunday-first format as well
+                const firstDay = startOfMonth.subtract(startOfMonth.day() === 0 ? 6 : startOfMonth.day() - 1, 'days')
+                const lastDay = endOfMonth.add(endOfMonth.day() === 0 ? 0 : 7 - endOfMonth.day(), 'days')
+                const weeks = props.weeks ?? lastDay.diff(firstDay, 'week') + 1
+                const showLeftMonth = month === 0
+                const showRightMonth = month + 1 === months
+
+                return (
+                    <table className="LemonCalendar__month" key={month}>
+                        <thead>
+                            <tr>
+                                {showLeftMonth && (
+                                    <th>
+                                        <LemonButton
+                                            status="stealth"
+                                            fullWidth
+                                            onClick={() => {
+                                                const newDate = dayjs(firstMonth)
+                                                    .subtract(1, 'month')
+                                                    .format('YYYY-MM-DD')
+                                                setFirstMonth(newDate)
+                                                props.onFirstMonthChanged?.(newDate)
+                                            }}
+                                        >
+                                            <IconChevronLeft />
+                                        </LemonButton>
+                                    </th>
+                                )}
+                                <th colSpan={7 - (showLeftMonth ? 1 : 0) - (showRightMonth ? 1 : 0)}>
+                                    <LemonButton
+                                        status="muted"
+                                        fullWidth
+                                        center
+                                        className="text-xs font-bold text-muted uppercase cursor-default"
+                                    >
+                                        {startOfMonth.format('MMMM')} {startOfMonth.year()}
+                                    </LemonButton>
+                                </th>
+                                {showRightMonth && (
+                                    <th>
+                                        <LemonButton
```

Comment:
Also, because those month buttons are larger (specifically, wider), than day buttons, and everything is in one table, Mondays of the leftmost month and Sundays of the rightmost month are oddly wider than other days:
<img width="93" alt="Screen Shot 2022-08-31 at 12 53 46" src="https://user-images.githubusercontent.com/4550621/187664386-a96d0e13-6df4-4179-91f3-2a8a64f48ad1.png">
I propose to take the month buttons + name out of the table and just make that part of the layout a flexbox.